### PR TITLE
Add data disk tutorial

### PR DESCRIPTION
**Pull request by balloob**
This adds a data disk tutorial. It re-uses existing flows as much as possible. To do this, I had to break up the CM4 installation procedure into a "disassemble" and "install CM4" procedure.

[Preview](https://deploy-preview-77--home-assistant-yellow.netlify.app/guides/add-ssd-existing-installation/)

I miss a screenshot for the move to data disk (don't have an SSD here). I think that the text could also use some love.

Things that could be improved:
 - Better screenshot for "Gathering required items"
 - Mention to power it on after putting case back together
 - Screenshot of move to data disk dialog
 - Final step? Clarify how long it takes. How to verify it works?

### CODE REVIEW

File: src/procedures/move-data-disk.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+              /></a>
+              <br><br>
+              Alternative, you can go to the storage page via Configuration, System, Storage.
+            </li>
+          </ul>
+      - title: Open the Move to Datadisk dialog
+        image: ha-config-storage-overflow-menu.png
+        content: |
+          <ul>
+            <li>Open the overflow menu by clicking on X in the top-right of the screen.</li>
```
Comment (agners) [inline src/procedures/move-data-disk.html:27]: Can we use the UTF-8 symbol ⋮ safely for the three dot menu thing? It seems to work here.,,

```suggestion
            <li>Open the overflow menu by clicking on ⋮ in the top-right of the screen.</li>
```

It seems we use "three dots" in the official documentation.

@frenck thoughts?

**Review by agners — CHANGES_REQUESTED** [review]
I think at the very least we should add the screenshot of the SSD selection before merging this.

There is one in the official documentation:
https://github.com/home-assistant/home-assistant.io/blob/bcbeabd542618cf2375d8d42c078ba8d3e78008d/source/images/screenshots/move-datadisk.png

However, there it is a S-ATA based SSD which appears as `/dev/sda`

I can also take one with `/dev/nvme0n1`.

File: src/procedures/move-data-disk.html (hunk 1 above)
Comment (balloob) [reply src/procedures/move-data-disk.html:27]: oh that was a placeholder. I can just get the SVG

Comment (balloob) [reply src/procedures/move-data-disk.html]: I added the SVG icon so it's very clear now.

**Comment by balloob** [comment]
Please make a new screenshot with /dev/nvme0n1

File: src/procedures/move-data-disk.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" style="width: 24px"><path d="M12,16A2,2 0 0,1 14,18A2,2 0 0,1 12,20A2,2 0 0,1 10,18A2,2 0 0,1 12,16M12,10A2,2 0 0,1 14,12A2,2 0 0,1 12,14A2,2 0 0,1 10,12A2,2 0 0,1 12,10M12,4A2,2 0 0,1 14,6A2,2 0 0,1 12,8A2,2 0 0,1 10,6A2,2 0 0,1 12,4Z" /></svg>
+              in the top-right of the screen.
+            </li>
+            <li>Select the <b>Move datadisk</b> option</li>
+          </ul>
+      - title: Moving to datadisk
+        image: select-new-datadisk.png
+        content: |
+          <ul>
+            <li>Select your hard drive and click on <b>MOVE</b>.</li>
```
Comment (balloob) [inline src/procedures/move-data-disk.html:38]: If it's always the same hard drive path, can we put it here.

File: src/procedures/move-data-disk.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            <li class="info">This can take a long time. Please be patient.
```
Comment (balloob) [inline src/procedures/move-data-disk.html:39]: I guess 4 minutes is not that long.

Comment (agners) [reply src/procedures/move-data-disk.html:39]: A couple of releases ago I optimized it to only copy the effective files (before it was raw dd afaik). With that its quite fast most of the time.

Maybe reorder, first bring the 4 minute example,, and then this point and soften a bit "fairly long time"?

**Pull request merged**